### PR TITLE
Add some missing punctuation marks

### PR DESCRIPTION
**Pull request by goyalankit**
While reading the documentation, it took me some time to understand the first two points. Modified it a bit to be more legible.

### CODE REVIEW

File: content/en/docs/components/training/tftraining.md [hunk 1]
```diff
@@ -432,12 +432,12 @@ A TFJob has a TFJobStatus, which has an array of TFJobConditions through which t
 
 Success or failure of a job is determined as follows
 
-* If a job has a **chief** success or failure is determined by the status
+* If a job has a **chief**, success or failure is determined by the status
   of the chief.
-* If a job has no chief success or failure is determined by the workers.
-* In both cases the TFJob succeeds if the process being monitored exits
+* If a job has no chief, success or failure is determined by the workers.
+* In both the cases, the TFJob succeeds if the process being monitored exits
```
Comment (8bitmp3) [inline content/en/docs/components/training/tftraining.md:438]:
```suggestion
* In both cases the TFJob succeeds if the process being monitored exits
```

"In both the cases" may be grammatically incorrect. I recommend reverting this particular change.

Thanks for the other comma suggestions.

cc @RFMVasconcelos @Bobgy

Comment (goyalankit) [reply content/en/docs/components/training/tftraining.md]: done.

**Comment by 8bitmp3** [comment]
/lgtm

thanks!

for approvals:

/assign @animeshsingh @Bobgy @joeliedtke

**Comment by Bobgy** [comment]
The change looks good to me, but why OWNERS is missing for training documentation?
/cc @kubeflow/wg-training-leads

**Pull request merged**